### PR TITLE
metadata creation

### PR DESCRIPTION
**Pull request by feki-rihab**

### CODE REVIEW

File: Dataset/CY_Pilot/consumer_reports_metadata.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    ""
+  ],
+  "keyword": [
+    "water consumer",
+    "report",
+    "water issues",
+    "location",
+    "water problem type"
+  ],
+  "publisher": "WDME",
```
Comment (flopezag) [inline Dataset/CY_Pilot/consumer_reports_metadata.json:28]: Let's as it is but here we should put the details of the partner that provide this dataset

File: Dataset/CY_Pilot/consumer_reports_metadata.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  "spatial": [],
+  "temporal": [],
+  "theme": [
+    "Environment",
+    "Water"
+  ],
+  "accessRights": "",
+  "creator": [
+    "CY Pilot"
```
Comment (flopezag) [inline Dataset/CY_Pilot/consumer_reports_metadata.json:37]: Idem

File: Dataset/CY_Pilot/consumer_reports_metadata.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  "qualifiedAttribution": [
+    ""
+  ],
+  "qualifiedRelation": [
+    ""
+  ],
+  "relation": [
+    ""
+  ],
+  "issued": "1983-07-16T12:51:26Z",
```
Comment (flopezag) [inline Dataset/CY_Pilot/consumer_reports_metadata.json:76]: I do not believe that this dataset has been issued on that date...

File: Dataset/CY_Pilot/consumer_reports_metadata.json [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  "version": "",
+  "versionNotes": [],
+  "wasGeneratedBy": [
+    "wdme.com"
+  ],
+  "license": "Apache-2.0",
+  "geolocation": false,
+  "addressAvailable": false,
+  "standardization": "Level3Standardization",
+  "standardizationSource": "",
```
Comment (flopezag) [inline Dataset/CY_Pilot/consumer_reports_metadata.json:94]: If we have Level3 or Level4, then we need to provide the link to the standardization body (URL) here.

**Review by flopezag — CHANGES_REQUESTED** [review]

File: Dataset/CY_Pilot/consumer_reports_metadata.json (hunk 4 above)
Comment (feki-rihab) [reply Dataset/CY_Pilot/consumer_reports_metadata.json:94]: Regarding this point, where can I find this standardization body (URL) ?

Comment (flopezag) [reply Dataset/CY_Pilot/consumer_reports_metadata.json:94]: It depends on each case, so if we have no details about the data that we have, I suggest to change to Level2Standardization due to in that case we do not need to put anything in standardizationSource.

**Comment by feki-rihab** [comment]
Modifications resolved.

**Pull request merged**